ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/privacy/methods/priv/PrivGetTransactionCount.java

```diff
@@ -45,14 +62,48 @@ public String getName() {
 
   @Override
   public JsonRpcResponse response(final JsonRpcRequest request) {
-    if (request.getParamLength() != 2) {
-      return new JsonRpcErrorResponse(request.getId(), JsonRpcError.INVALID_PARAMS);
-    }
-
     final Address address = parameters.required(request.getParams(), 0, Address.class);
-    final String privacyGroupId = parameters.required(request.getParams(), 1, String.class);
```

Comment:
By moving this line into the try/catch block you change the error condition behaviour, instead of a `InvalidJsonRpcParameters` being thrown, it will be caught by the `catch` and you will receive `sonRpcError.GET_PRIVATE_TRANSACTION_NONCE_ERROR` ...is that the intention?